[
{"message":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","level":"FATAL","type":"PLAYGROUND","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","log":"/tmp/tmp.DPn4wcpjkW"}
,
{"message":"openssl_pkey_export(): cannot get key from parameter 1 in /wordpress/wp-content/plugins/oauth2-provider/includes/functions.php on line 376","level":"WARNING","type":"PHP","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:22 UTC] PHP Warning:  openssl_pkey_export(): cannot get key from parameter 1 in /wordpress/wp-content/plugins/oauth2-provider/includes/functions.php on line 376","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"openssl_pkey_get_details() expects parameter 1 to be resource, bool given in /wordpress/wp-content/plugins/oauth2-provider/includes/functions.php on line 381","level":"WARNING","type":"PHP","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:22 UTC] PHP Warning:  openssl_pkey_get_details() expects parameter 1 to be resource, bool given in /wordpress/wp-content/plugins/oauth2-provider/includes/functions.php on line 381","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Trying to access array offset on value of type null in /wordpress/wp-content/plugins/oauth2-provider/includes/functions.php on line 382","level":"NOTICE","type":"PHP","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:22 UTC] PHP Notice:  Trying to access array offset on value of type null in /wordpress/wp-content/plugins/oauth2-provider/includes/functions.php on line 382","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:33 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_refresh_tokens MODIFY refresh_token VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(358): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_refresh_tokens MODIFY refresh_token VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:33 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_refresh_tokens MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(359): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_refresh_tokens MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:33 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_public_keys MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(361): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_public_keys MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:33 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_jwt MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(362): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_jwt MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:33 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_authorization_codes MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(363): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_authorization_codes MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown show type: KEYS FROM.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:33 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW KEYS FROM wp_oauth_access_tokens</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: KEYS FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SHOW KEYS FROM ...')\n#2 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(375): wpdb->get_results('SHOW KEYS FROM ...')\n#3 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#7 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#8 {main}</pre>\n for query SHOW KEYS FROM wp_oauth_access_tokens made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:33 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_access_tokens MODIFY access_token VARCHAR(1000);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(383): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_access_tokens MODIFY access_token VARCHAR(1000); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: ap_generated.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:33 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_access_tokens ADD `ap_generated` VARCHAR( 32 ) AFTER \t`scope`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_oauth_access_tokens, `AP_GENERATED`, varchar(32)</li>\n<li>Executing: ALTER TABLE wp_oauth_access_tokens ADD `ap_generated`   text  \t | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: ap_generated.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(391): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_access_tokens ADD `ap_generated` VARCHAR( 32 ) AFTER \t`scope` made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:33 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge` VARCHAR( 1000 ) AFTER `id_token`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_oauth_authorization_codes, `CODE_CHALLENGE`, varchar(1000)</li>\n<li>Executing: ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge`   text   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(401): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge` VARCHAR( 1000 ) AFTER `id_token` made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge_method.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:33 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge_method` VARCHAR( 32 ) AFTER `id_token`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_oauth_authorization_codes, `CODE_CHALLENGE_METHOD`, varchar(32)</li>\n<li>Executing: ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge_method`   text   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge_method.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(411): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge_method` VARCHAR( 32 ) AFTER `id_token` made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:43 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_refresh_tokens MODIFY refresh_token VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(358): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_refresh_tokens MODIFY refresh_token VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:43 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_refresh_tokens MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(359): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_refresh_tokens MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:43 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_public_keys MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(361): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_public_keys MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:43 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_jwt MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(362): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_jwt MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:43 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_authorization_codes MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(363): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_authorization_codes MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown show type: KEYS FROM.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:43 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW KEYS FROM wp_oauth_access_tokens</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: KEYS FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SHOW KEYS FROM ...')\n#2 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(375): wpdb->get_results('SHOW KEYS FROM ...')\n#3 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#7 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#8 {main}</pre>\n for query SHOW KEYS FROM wp_oauth_access_tokens made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:43 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_access_tokens MODIFY access_token VARCHAR(1000);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(383): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_access_tokens MODIFY access_token VARCHAR(1000); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: ap_generated.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:43 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_access_tokens ADD `ap_generated` VARCHAR( 32 ) AFTER \t`scope`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_oauth_access_tokens, `AP_GENERATED`, varchar(32)</li>\n<li>Executing: ALTER TABLE wp_oauth_access_tokens ADD `ap_generated`   text  \t | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: ap_generated.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(391): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_access_tokens ADD `ap_generated` VARCHAR( 32 ) AFTER \t`scope` made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:43 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge` VARCHAR( 1000 ) AFTER `id_token`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_oauth_authorization_codes, `CODE_CHALLENGE`, varchar(1000)</li>\n<li>Executing: ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge`   text   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(401): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge` VARCHAR( 1000 ) AFTER `id_token` made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge_method.","level":"FATAL","type":"SQL","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:43 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge_method` VARCHAR( 32 ) AFTER `id_token`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_oauth_authorization_codes, `CODE_CHALLENGE_METHOD`, varchar(32)</li>\n<li>Executing: ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge_method`   text   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge_method.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(411): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge_method` VARCHAR( 32 ) AFTER `id_token` made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method OBJ_MTL_Viewer::register_styles() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324","level":"FATAL","type":"PHP","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:58:53 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method OBJ_MTL_Viewer::register_styles() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-settings.php(727): do_action('init')\n#3 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#4 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#5 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#6 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method OBJ_MTL_Viewer::register_styles() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324","level":"FATAL","type":"PHP","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:59:02 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method OBJ_MTL_Viewer::register_styles() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-settings.php(727): do_action('init')\n#3 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#4 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#5 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#6 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
,
{"message":"Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method OBJ_MTL_Viewer::register_styles() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324","level":"FATAL","type":"PHP","test":"jspi-boot","plugin":"obj-mtl-viewer","details":"[01-Jun-2025 06:59:12 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method OBJ_MTL_Viewer::register_styles() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#2 /wordpress/wp-settings.php(727): do_action('init')\n#3 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')\n#4 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')\n#5 /internal/eval.php(3): require_once('/wordpress/wp-l...')\n#6 {main}\n  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"obj-mtl-viewer\"},\"options\":{\"activate\":true}}) : PHP.run() failed with exit code 255 and the following output: \n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      ... 2 lines matching cause stack trace ...\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: \n        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:73:13)\n        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)\n        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)\n        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)\n        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n      response: PHPResponse {\n        httpStatusCode: 500,\n        headers: {\n          'x-powered-by': [ 'PHP/8.0.30-dev' ],\n          'content-type': [ 'text/html; charset=UTF-8' ],\n          expires: [ 'Wed, 11 Jan 1984 05:00:00 GMT' ],\n          'cache-control': [ 'no-cache, must-revalidate, max-age=0, no-store, private' ]\n        },\n        bytes: Uint8Array(2653) [\n           60,  33,  68,  79,  67,  84,  89,  80,  69,  32, 104, 116,\n          109, 108,  62,  10,  60, 104, 116, 109, 108,  32, 108,  97,\n          110, 103,  61,  34, 101, 110,  45,  85,  83,  34,  62,  10,\n           60, 104, 101,  97, 100,  62,  10,   9,  60, 109, 101, 116,\n           97,  32, 104, 116, 116, 112,  45, 101, 113, 117, 105, 118,\n           61,  34,  67, 111, 110, 116, 101, 110, 116,  45,  84, 121,\n          112, 101,  34,  32,  99, 111, 110, 116, 101, 110, 116,  61,\n           34, 116, 101, 120, 116,  47, 104, 116, 109, 108,  59,  32,\n           99, 104,  97, 114,\n          ... 2553 more items\n        ],\n        exitCode: 255,\n        errors: ''\n      },\n      source: 'request'\n    }\n  }\n}\n\n","log":"/logs/plugins/o/obj-mtl-viewer/error.json"}
]
